src/kioskboard.js

```diff
@@ -736,7 +741,7 @@
             var docTop = window.document.documentElement.scrollTop || 0;
             var inputThreshold = isPlacementTop ? (theInput.clientHeight + 20) : 50;
             var theInputOffsetTop = Math.round(inputTop + docTop) - inputThreshold;
-            var isPaddingTop = (theInputOffsetTop < keyboardHeight) && isPlacementTop;
+            var isPaddingTop = theInputOffsetTop < keyboardHeight;
```

Comment:
This is removing the latest fix that you have made with https://github.com/furcan/KioskBoard/pull/52